### PR TITLE
Edit survey feature #37

### PR DESCRIPTION
**Pull request by MikeEvans6**
Need to add MySurveyControllerTests still

### CODE REVIEW

File: src/test/java/application/models/TestEditDTO.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
```
Comment (cyang030) [inline src/test/java/application/models/TestEditDTO.java]: "import org.junit.jupiter.api.Assertions.assertEquals" for Junit5

File: src/test/java/application/models/TestEditDTO.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package application.models;
+
+import org.junit.Before;
+import org.junit.Test;
```
Comment (cyang030) [inline src/test/java/application/models/TestEditDTO.java]: import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

**Review by cyang030 — APPROVED** [review]

**Review by thomas7carriere — APPROVED** [review]
Great, MS2 done

**Comment by MikeEvans6** [comment]
close #37

**Pull request merged**